front/src/applications/osrd/components/Simulation/Allowances/StandardAllowanceDefault.js

```diff
@@ -162,56 +191,88 @@ export default function StandardAllowanceDefault(props) {
     }
   }, [trainDetail]);
 
+  useEffect(() => {
+    if(providedType) setValue(providedType)
+  }, [providedType]);
+
   return (
     <div className="row w-100 mareco">
-      <div className="col-md-2 text-normal">{t('sandardAllowancesWholePath')}</div>
-      <div className="col-md-1 text-normal">{t('Valeur par défault')}</div>
-      <div className="col-md-3">
-        <SelectSNCF
-          id="distributionTypeSelector"
-          options={distributionsTypes}
-          labelKey="label"
-          onChange={handleDistribution}
-          sm
-          value={distribution}
-        />
-      </div>
-      <div className="col-md-3">
-        <InputGroupSNCF
-          id="standardAllowanceTypeSelect"
-          options={allowanceTypes}
-          handleType={handleType}
-          value={value.value}
-          type={value.type}
-          sm
-        />
-      </div>
-      <div className="col-md-3">
-        <button
-          type="button"
-          onClick={addStandard}
-          className={`btn btn-success btn-sm mr-1 ${value.value === 0 ? 'disabled' : null}`}
-        >
-          {t('apply')}
-        </button>
-        <button
-          type="button"
-          onClick={() => delStandard(value)}
-          className={`btn btn-danger btn-sm ${value.value === 0 ? 'disabled' : null}`}
-        >
-          <FaTrash />
-        </button>
+      <div className="col-md-2 text-normal">{title || t('sandardAllowancesWholePath')}</div>
+
+      <div className="col">
+      <div className="row">
+          {options.setDistribution && (
+            <React.Fragment>
+            <div className="col-md-2 text-normal">{t('Valeur par défault')}</div>
```

Comment:
When using locale, you have to use english and concise text in camelCase.
`t('defaultValue')` for example.

Reply:
done